V1

maxproway opened the pull request:
[CLONED REPO](https://github.com/maxproway/lp-nothing)
[DEMO LINK](https://maxproway.github.io/lp-nothing/)

volodymyr-soltys97 reviewed (changes requested):
Good job 👍 
Let's improve your page
1. Remove this link from the PR description 
<img width="184" alt="image" src="https://github.com/user-attachments/assets/268fad45-9974-4375-9b15-df39aa81b0ff">

2. You need to show phone number when hovering this icon, check it on the layout 
<img width="92" alt="image" src="https://github.com/user-attachments/assets/ed2c1eb7-bedd-4276-b1fd-108ee18b5d79">

3. Fix the price position for these blocks, check it
<img width="1108" alt="image" src="https://github.com/user-attachments/assets/260ca6f3-c240-43b9-9208-792eff6017dd">

4. The user cannot be able to change the textarea field
<img width="1080" alt="image" src="https://github.com/user-attachments/assets/7cc6108d-4dae-414b-adbf-5d91565a730d">

5. Make all these fields are required before sending 
<img width="555" alt="image" src="https://github.com/user-attachments/assets/9d75abd4-b684-4a85-978a-e316c1d0a595">

6. Change the autofill styles for the form fields
https://css-tricks.com/snippets/css/change-autocomplete-styles-webkit-browsers/ 
<img width="556" alt="image" src="https://github.com/user-attachments/assets/addb8bb3-82e3-44af-b2fb-2dbb0f4a6682">

7. The font styles must be are same everywhere 
<img width="155" alt="image" src="https://github.com/user-attachments/assets/7bffea0a-a1cb-403e-8486-54c1e5c4276d">

maxproway commented:
Thank you, I've fixed all except p6. "autofill styles". I'm not sure how it should work. In my case autofill always stay with a yellow background on my side. Here the code what I added to my input and textarea:
`    &:-webkit-autofill,
    &:-webkit-autofill:hover,
    &:-webkit-autofill:focus {
      background-color: $color-light-background;
    }`

Even the reffered codepen (https://codepen.io/team/css-tricks/pen/oxyJxR) is not working for me, background still stays yellow.

volodymyr-soltys97 reviewed (changes requested):
GJ!
To improve:
1. This comment stills not fixed from the previous review, user cannot be able to change size on textarea 
<img width="893" alt="image" src="https://github.com/user-attachments/assets/e285e48b-23a1-41e9-9010-60a5fa91dbe8">

2. Not fixed 
<img width="778" alt="image" src="https://github.com/user-attachments/assets/1e0f115d-7c97-4eba-a254-9b3db81844f8">

3. Not fixed 
<img width="454" alt="image" src="https://github.com/user-attachments/assets/724ec67c-5fa4-45a8-a4fe-d06595a35f2a">

If you need help feel free in the fe_chat

maxproway commented:
I probably didn't made a npm deploy after changes. Placeholders and Texarea fixed size should be ok. But I still frustrating with autofills...

maxproway commented:
Probably the issue also in my browser extension  (Dashlane).
I've added following code to my inputs and textareas:
```
    &:-webkit-autofill,
    &:-webkit-autofill:hover,
    &:-webkit-autofill:focus {
      -webkit-box-shadow: 0 0 0px 1000px $color-light-background inset;
    }
```

please check:

anastasiiavorobiova reviewed (approved):
Great work!
